src/atomicui/organisms/SearchBox/SearchBox.tsx

```diff
@@ -339,12 +348,49 @@ const SearchBox: React.FC<SearchBoxProps> = ({
 									))}
 								</Flex>
 							),
-							Empty:
+							Empty: !isNLPChecked ? (
 								!!value && !suggestions?.length ? (
 									<Flex className="not-found-container">
 										<NotFoundCard />
 									</Flex>
 								) : null
+							) : (
+								<Flex
+									gap={0}
+									alignItems="start"
+									className="inner-search-component"
+									// marginLeft="10px"
+									style={{
+										flexDirection: "column",
+										gap: "0"
+									}}
+								>
+									<h3>Try Asking:</h3>
+									<Flex
+										style={{
+											flexDirection: "column",
+											gap: "0.1px",
+											marginBottom: "12px",
+											fontStyle: "italic",
+											fontSize: "15px"
+										}}
+									>
+										<q>What are some good places to get coffee in Boston?</q>
+										<q>Where can I get a haircut?</q>
+										<q>Whats open now for dinner?</q>
```

Comment:
Missing single quote: What's open now for dinner?. 
In addition to that, this is not a query type we support fully. LLM will search for dinner restaurants but we don't know which ones are open. I suggest we use queries we fully support.